### ISSUE TITLE
Request for pretrainig data label

### ISSUE THREAD

**Issue by purplerainf** [hspark1212/MOFTransformer#172]
Dear authors,

Thank you for developing and sharing this great model and data. 

I'm interested in reproducing your pretraining and fine-tuning procedures. I have found most of the data from other github issues, except the pretraining data label. 

If possible, could you please share the pretrainig data label via email, like others? my email address is: purplerainf@gmail.com

Bests,

**Comment by Yeonghun1675** [comment]
Hi @purplerainf ,
The label of pre-trainning data is in our FigShare (http://figshare.com/articles/dataset/PMTransformer_pre-trained_model/22698655).
You can download `jsons.zip` and use the each pre-training label (vfp, moc, and mtp)

**Comment by purplerainf** [comment]
Thank you very much for your sharing!!
